packages.json

```diff
@@ -18967,6 +18967,169 @@
     "license": "MIT",
     "web": "http://github.com/jayvanderwall/desim"
   },
+  {
+    "name": "NimpleHTTPServer",
+    "url": "https://github.com/Hydra820/NimpleHTTPServer",
+    "method": "git",
+    "tags": [
+      "Simple",
+      "HTTP",
+      "Server"
+    ],
+    "description": "SimpleHTTPServer module based on net sockets",
+    "license": "HYDRA",
+    "web": "https://github.com/Hydra820/NimpleHTTPServer"
+  },
+  {
+    "name": "nim-bitarray",
```

Comment:
this is wrong, it should be `bitarray`. Dashes should generally not be allowed in package names.